editor.planx.uk/src/pages/PlatformAdminPanel/utils.ts

```diff
@@ -1,15 +1,13 @@
+import { capitalize } from "lodash";
 import { AdminPanelData } from "types";
 
-export const internalTeamNames = [
-  "WikiHouse",
-  "PlanX",
-  "Open Systems Lab",
-  "Testing",
-  "Open Digital Planning",
-  "Environment Agency",
```

Comment:
This original list filtered out the EA, but I'm not omitting them in this update - they may actually launch services that we want to see in the admin panel !